ThaDeveloper/Fast-Food-Fast
# 161126324 Post order

#### What does this PR do?
Have post order route
#### Description of Task to be completed?
Post order data to the API
The user needs to be logged in
#### How should this be manually tested?

- `$ git clone https://github.com/ThaDeveloper/Fast-Food-Fast`
- `$ cd Fast-Food-Fast`
- Open index.html in your favorite browser.
- Sign in and add items to cart, then check at order page

#### Any background context you want to provide?
Built in : Vanilla Javascript
#### What are the relevant pivotal tracker stories?
[#161126324](https://www.pivotaltracker.com/story/show/161126324)
